package.json

```diff
@@ -1,120 +1,50 @@
 {
-  "name": "@reedsy/quill",
-  "version": "2.0.0-reedsy-4.0.1",
-  "description": "Your powerful, rich text editor",
+  "name": "quill-monorepo",
+  "version": "2.0.0-rc.0",
```

Comment:
I don't think this should change

Reply:
It's been [moved](https://github.com/reedsy/quill/commit/9df0daee3b6197ea97ff8c54d13e46844fc989ef#diff-196802e154b2cb49debe13bfd545542edf12dd7d90ef32b3c3816ab2c66df42aR2-R3), although I didn't rename the `name` field down there 👍🏼